root/static/styles/layout.less

```diff
@@ -18,6 +18,10 @@ body {
 a {
     color:@link-default;
     text-decoration: none;
+    
+    &#sidebar {
```

Comment:
That matches `a#sidebar`, i.e. a link with `id="sidebar"`. :) You need an `a` rule inside the `#sidebar` block instead.